go.mod

```diff
@@ -357,7 +357,7 @@ require (
 	github.com/tdakkota/asciicheck v0.4.0 // indirect
 	github.com/telepresenceio/watchable v0.0.0-20220726211108-9bb86f92afa7 // indirect
 	github.com/tetafro/godot v1.4.20 // indirect
-	github.com/tetratelabs/func-e v1.1.5-0.20240822223546-c85a098d5bf0 // indirect
+	github.com/tetratelabs/func-e v1.1.5-0.20250319211008-ca8702e12788 // indirect
```

Comment:
this is to use https://github.com/tetratelabs/func-e/pull/458 and will be compatible with https://github.com/envoyproxy/gateway/pull/5527